### PR TITLE
card classes added to home page

### PR DESCRIPTION
**Pull request by murex971**
Signed-off-by: Nupur Agrawal <nupur202000@gmail.com>

Before submitting pull request, please review the following checklist:

- [ ] Make sure you have read our [CONTRIBUTING.md](https://github.com/phpmyadmin/phpmyadmin/blob/master/CONTRIBUTING.md) document.
- [ ] Make sure you are making a pull request against the correct branch. For example, for bug fixes in a released version use the corresponding QA branch and for new features use the _master_ branch. If you have a doubt, you can ask as a comment in the bug report or on the mailing list.
- [ ] Every commit has proper `Signed-off-by` line as described in our [DCO](https://github.com/phpmyadmin/phpmyadmin/blob/master/DCO). This ensures that the work you're submitting is your own creation.
- [ ] Every commit has a descriptive commit message.
- [ ] Every commit is needed on its own, if you have just minor fixes to previous commits, you can squash them.
- [ ] Any new functionality is covered by tests.

### CODE REVIEW

**Review by williamdes — APPROVED** [review]
LGTM

**Comment by mauriciofauth** [comment]
@sudo-bot Deploy pull request

**Review by MauricioFauth — CHANGES_REQUESTED** [review]
Can you rebase with _master_ branch and fix the conflicts?

File: templates/home/index.twig [hunk 1]
```diff
@@ -69,134 +72,142 @@
...
-                </a>
-              </li>
-            </ul>
+                <li id="li_pma_support">
+                  <a href="{{ 'https://www.phpmyadmin.net/support/'|link }}" target="_blank" rel="noopener noreferrer">
+                    {% trans 'Get support' %}
+                  </a>
+                </li>
+                <li id="li_pma_changes">
+                  <a href="changelog.php{{ get_common() }}" target="_blank">
```
Comment (mauriciofauth) [inline templates/home/index.twig]:
```suggestion
                  <a href="{{ url('/changelog') }}" target="_blank">
```

File: templates/home/index.twig [hunk 2]
```diff
@@ -69,134 +72,142 @@
...
+                    {% trans 'List of changes' %}
+                  </a>
+                </li>
+                <li id="li_pma_license">
+                  <a href="license.php{{ get_common() }}" target="_blank">
```
Comment (mauriciofauth) [inline templates/home/index.twig]:
```suggestion
                  <a href="{{ url('/license') }}" target="_blank">
```

File: templates/preferences/manage/main.twig [hunk 3]
```diff
@@ -29,36 +30,36 @@
                         <div class="localStorage-exists">
                             {% trans 'Saved on: @DATE@' %}
                         </div>
-                        <div class="localStorage-empty">
-                            {{ 'You have no saved settings!'|trans|notice }}
```
Comment (mauriciofauth) [inline templates/preferences/manage/main.twig]: You removed the `localStorage-empty` `div`.

File: templates/preferences/manage/main.twig [hunk 4]
```diff
@@ -81,33 +82,36 @@
...
-                {{ get_hidden_inputs() }}
-                {% trans 'You can reset all your settings and restore them to default values.' %}
-                <br><br>
-                <input class="btn btn-secondary" type="submit" name="submit_clear" value="{% trans 'Reset' %}">
-            </form>
+        <div class="group card">
+            <h2 class="card-header">{% trans 'Reset' %}</h2>
+            <div class="card-body">
+                <form class="group-cnt prefs-form disableAjax" name="prefs_reset"
+                      action="prefs_manage.php" method="post">
```
Comment (mauriciofauth) [inline templates/preferences/manage/main.twig]:
```suggestion
                      action="{{ url('/preferences/manage') }}" method="post">
```

File: templates/preferences/two_factor/main.twig [hunk 5]
```diff
@@ -1,60 +1,68 @@
...
-        <input class="btn btn-secondary" type="submit" name="2fa_remove" value="
-          {%- trans "Disable two-factor authentication" %}">
-      </form>
+  <div class="group card row">
+    <h2 class="card-header">{{ backend_name }}</h2>
+    <div class="card-body">
+      <div class="group-cnt">
+        <p>{% trans "You have enabled two factor authentication." %}</p>
+        <p>{{ backend_description }}</p>
+        <form method="post" action="prefs_twofactor.php">
```
Comment (mauriciofauth) [inline templates/preferences/two_factor/main.twig]:
```suggestion
        <form method="post" action="{{ url('/preferences/twofactor') }}">
```

File: templates/preferences/two_factor/main.twig [hunk 6]
```diff
@@ -1,60 +1,68 @@
...
-            <p>{{ backend["description"] }}</p>
-          </label>
-        {% endfor %}
-        <input class="btn btn-secondary" type="submit" value="{% trans "Configure two-factor authentication" %}">
-      </form>
+  <div class="group card row">
+    <h2 class="card-header">{% trans "Configure two-factor authentication" %}</h2>
+    <div class="card-body">
+      <div class="group-cnt">
+        <form method="post" action="prefs_twofactor.php">
```
Comment (mauriciofauth) [inline templates/preferences/two_factor/main.twig]:
```suggestion
        <form method="post" action="{{ url('/preferences/twofactor') }}">
```

File: themes/pmahomme/scss/_common.scss [hunk 7]
```diff
@@ -2775,7 +2775,6 @@ fieldset {
 }
 
 .prefsmanage_opts {
-  margin-#{$left}: 2em;
 }
```
Comment (mauriciofauth) [inline themes/pmahomme/scss/_common.scss]: You should remove the empty block.
See: https://travis-ci.org/phpmyadmin/phpmyadmin/jobs/575149921#L415

**Review by MauricioFauth — CHANGES_REQUESTED** [review]

**Pull request closed without merging**